packages/cli/src/utils/platform.ts

```diff
@@ -146,27 +146,77 @@ export const configDirectoryPath = new XdgAppPaths('prisma-platform-cli').config
 export const authConfigPath = path.join(configDirectoryPath, 'auth.json')
 
 export async function writeAuthConfig(data: AuthConfig) {
-  await fs.mkdirp(configDirectoryPath)
-
-  return await fs.writeJSON(authConfigPath, data)
+  try {
+    await fs.mkdirp(configDirectoryPath)
+    return await fs.writeJSON(authConfigPath, data)
+  } catch (error) {
+    debug('Error from writeAuthConfig()', error)
+    return error as Error
```

Comment:
nit: This could be done differently
```suggestion
  } catch (error: Error) {
    debug('Error from writeAuthConfig()', error)
    return error
```

Reply:
True! Tho both ways are a bit bad 🙈 Better way would be to deal with unknown type and if it does not extend error, then convert it to an error

Reply:
A source of problems could be that if we have mixed behaviors throw vs return an Error we can easily forget to handle them?